clients/imodelhub/src/IModelCloudEnvironment.ts

```diff
@@ -14,8 +14,7 @@ import { IModelClient } from "./IModelClient";
  * @internal
  */
 export interface ContextManagerClient {

```

Comment:
Should this be renamed to (along with the description above) ?

Reply:
I agree. Currently these kind of semantic changes with names and descriptions is intended for a future PR as it will be very widespread and require attention about if it will cause problematic breaking changes to the API.

Reply:
Ok, I'll stop mentionning those then.